lib/NodeHelpers.js

```diff
@@ -131,17 +160,24 @@ function getLocalMatrix4(node, result) {
     rotation.w = rotationArray[3];
 
     return Matrix4.fromTranslationQuaternionRotationScale(translation, rotation, scale, result);
-}
+};
 
 var packedParametersScratch = {
     meshes : undefined,
     parameters : undefined,
     primitiveFunction : undefined
 };
 
-// Perform an operation on each primitive in a scene.
-// PrimitiveFunction should expect primitive, a meshPrimitiveID, some parameters, and the node itself
-function forEachPrimitiveInScene(gltf, scene, primitiveFunction, parameters) {
+/**
+ * Perform an operation on each primitive in a scene.
+ * primitiveFunction has the signature: function(primitive, meshPrimitiveId, parameters, node).
+ *
+ * @param {Object} gltf A javascript object holding a glTF hierarchy.
+ * @param {Object} scene The glTF scene to traverse.
+ * @param {Function} primitiveFunction The function to apply to each primitive.
+ * @param {Object} parameters Parameters to pass to primitiveFunction.
+ */
+NodeHelpers.forEachPrimitiveInScene = function(gltf, scene, primitiveFunction, parameters) {
```

Comment:
Let's mark all classes that are not exposed in the public API as `@private` like we do in Cesium; otherwise, it is not obvious unless you look in index.js.